Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp

```diff
@@ -781,12 +781,14 @@ void NetworkConnectionToWebProcess::registerURLSchemesAsCORSEnabled(Vector<Strin
 
 static bool shouldTreatAsSameSite(const URL& firstParty, const URL& url)
 {
-#if PLATFORM(COCOA) || USE(SOUP)
+#if HAVE(COOKIE_CHANGE_LISTENER_API)
```

Comment:
I'm not a reviewer but I can add some comments to this PR.

I think using HAVE(COOKIE_CHANGE_LISTENER_API) looks natural because that was indicated according to cb077473952d7036cceedb1a721502d74ee766c5.